src/basis.jl

```diff
@@ -2,11 +2,14 @@
 # ------------------------------------------
 #   ACE Basis  
 
-export basis_params, degree_params, radbasis_params, transform_params
+export rpi_basis_params, pair_basis_params, degree_params, radbasis_params, transform_params
```

Comment:
Is there a Julia's rule of thumb on what should get exported and what should be called as `Module.function()`?

Reply:
we can choose our own rules, generally speaking anything user oriented and which doesn't clash with other packages should be exported I think.

Reply:
all the above look ok to me.

Reply:
Okay! May I suggest exporting `*_params()` and `generate_*()`? I think both a convenient way of generating the parameters and making stuff from them are equally useable.

Reply:
definitely export `*_params`. Why export `generate_*`? isn't that an internal thing?

Reply:
again as usual - if there is a good reason you as a user want this exported, I won't stand in the way. I'm just asking questions to make sure we are thinking this through.

Reply:
No, you're right, there shouldn't be a need to mess with any of the `generate_*`.